### ISSUE TITLE
Namespace error in Magento 1 EQP

### ISSUE THREAD

**Issue by devheidelpay** [magento/marketplace-eqp#49]
Hello magento,

we are getting the following error message on all of our classes and can't get the meaning when running.

vendor/bin/phpcs --standard=MEQP1 --ignore=vendor .

 19 | WARNING | Each class must be in a namespace of at least one
    |         | level (a top-level vendor name)

If we are not wrong, there are no namespace defined in magento 1 and if we define one for our class the autolader will not work any longer.

For example
```
<?php
class HeidelpayCD_Edition_Model_Payment_Abstract extends Mage_Payment_Model_Method_Abstract
{
...
}
```

Is this a miss behavior?

Thanks for help.

**Comment by lenaorobei** [comment]
@heidelpay do you use the latest version of coding standard? This issue was fixed in 14e1916a9f9f35b5668008873d76e4f0ec432069.

**Comment by devheidelpay** [comment]
@lenaorobei Can confirm this answer. We were on 1.0.4 and it is fixed in 1.0.5 .Thx for help.
